crates/sqltest/src/main.rs

```diff
@@ -327,7 +327,7 @@ async fn update_test_files(files: Vec<PathBuf>, engine: DbType, format: bool) ->
     Ok(())
 }
 
-/// Different from [`sqllogictest::update_test_file`], we re-implement it here to print some
+/// Different from [`sqllogictest`], we re-implement logic here to print some
 /// progress information.
```

Comment:
The original `impl` is not there, but this is fine, this only internal and nobody else use it.